packages/ui-app/src/InputFile.tsx

```diff
@@ -8,11 +8,12 @@ import { BareProps } from '@polkadot/ui-app/types';
 import React from 'react';
 import Dropzone from 'react-dropzone';
 
-import Labelled from './Labelled';
 import classes from './util/classes';
+import Labelled from './Labelled';
 import translate from './translate';
 
 type Props = BareProps & {
+  acceptedFormats?: string,
```

Comment:
I believe it should maybe be `acceptedFormat` since it is a string and singular? (Or rather I'm probably not seeing how to specify multiples then, i.e. would have expected `Array<string>` for multiples)

Reply:
If indeed valid, i.e. can accept `.json;.txt` (all in-one), maybe we should just leave at least a comment above.